### ISSUE TITLE
w3c validation error: An img element with no alt attribute must not have any aria-*

### ISSUE THREAD

**Issue by robertmausser** [adaptlearning/adapt-contrib-graphic#85]
Error An img element with no alt attribute must not have any aria-*
attributes other than aria-hidden .
From line 135, column 1793; to line 135, column 1943
`ontainer"><img class="component__image graphic__image" src="course/en
/images/icon_obj.png" aria-label="Placeholder graphic." ariahidden="
false" loading="eager"></div>`

**Comment by oliverfoster** [comment]
We use aria-label rather than alt.
Based on the computation presedence from the Accessible Name and Description computation algorithm https://www.w3.org/TR/accname-1.2/#mapping_additional_nd

It says that the presedence order is: aria-labelledby, aria-describedby, aria-label, then alt then title. Being that the alt attribute only works on the img tag and aria-label is supported in all screen readers on all elements, we use aria-label as its intent is more specific.
